sample/android/build.gradle.kts

```diff
@@ -1,13 +1,12 @@
 plugins {
-    id("com.mikepenz.convention.kotlin")
     id("com.mikepenz.convention.android-application")
+    id("com.mikepenz.convention.kotlin")
     id("com.mikepenz.convention.compose")
-    id("com.mikepenz.aboutlibraries.plugin")
```

Comment:
why the removal of the aboutlibraries plugin?

Reply:
oh. I see it's in the shared module. 

One note on that. Putting it in the shared module will miss out on including the dependencies from the application, resulting in attribution given for not the full set of dependencies. 

I think it would be better to have the plugin to generate the json in the app, while the shared only contains the ui for it.

Reply:
You're right, I'll try